portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -38,7 +38,7 @@
      */
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {        
-        int limit = Integer.parseInt(request.getParameter(Constant.COMMENTS_LIMIT));
+        int limit = Integer.parseInt(request.getParameter(Constants.COMMENTS_LIMIT));
```

Comment:
This doesn't look related to this PR and it looks more like a bug in #3. The right thing would be to fix this in a separate (small) PR.

Reply:
Ah, I see. Is this something to keep in mind for the future or should I try and move it to its own separate PR now?

Reply:
You can keep this change this PR but please indicate in the PR description the change. :-)